code/game/machinery/computer/arcade.dm

```diff
@@ -24,11 +23,33 @@
 							/obj/item/toy/prize/mauler						= 1,
 							/obj/item/toy/prize/odysseus					= 1,
 							/obj/item/toy/prize/phazon						= 1,
-							/obj/item/toy/waterflower						= 1,
-							/obj/random/action_figure						= 1,
-							/obj/random/plushie								= 1,
-							/obj/item/toy/cultsword							= 1
-							)
+							/obj/item/toy/waterflower						= 11,
+							/obj/random/action_figure						= 11,
+							/obj/random/plushie								= 55,
+							/obj/item/toy/cultsword							= 11,
+							/obj/item/toy/balloon							= 22,
+							/obj/item/toy/syndicateballoon					= 11,
+							/obj/item/toy/nanotrasenballoon					= 11,
+							/obj/item/toy/katana							= 11,
+							/obj/item/toy/bosunwhistle						= 11,
+							/obj/item/weapon/storage/belt/champion			= 11,
+							/obj/item/weapon/pen/invisible					= 11,
+							/obj/item/weapon/grenade/smokebomb				= 1,
+							/obj/item/weapon/bikehorn						= 11,
+							/obj/item/weapon/lipstick/random				= 11,
+							/obj/item/clothing/mask/fakemoustache			= 11,
+							/obj/item/clothing/mask/gas/clown_hat			= 11,
+							/obj/item/clothing/mask/gas/mime				= 11,
+							/obj/item/weapon/gun/energy/wand/toy			= 11,
+							/obj/item/weapon/gun/energy/mousegun			= 11,

```

Comment:
lol no mouse gun in an arcade machine.

Reply:
Removed. Didn't know what a mouse gun was until I got it in a gift.